packages/cli/src/lib/commands/up/index.ts

```diff
@@ -92,6 +96,15 @@ const up = async ({
   })
   const projectName = userSpecifiedProjectName ?? userModel.name
   const remoteDir = remoteProjectDir(projectName)
+
+  const exposedServices = getExposedTcpServices(userModel)
```

Comment:
it would probably simplify things if `getExposedTcpServices` returned just `[serviceName, portNumber]`, then you could do something like
```typescript
const env = Object.fromEntries(
  getExposedTcpServices(userMode)
  .map(([service, port]) => [
    `PREEVY_BASE_URI_${service.toUpperCase()}_${port}`, 
    tunnelUrl(...),
  ])
)
```